057-AzureRouteServer/Coach/README.md

```diff
@@ -0,0 +1,72 @@
+	 - Azure Route Server basic concepts, interaction with NVAs
+- Challenge 03: **[Connect Network Virtual Appliance to an SDWAN environment](./Solution-03.md)**
+	 - Azure Route Server interaction with Virtual Network Gateways (VPN or ExpressRoute)
+- Challenge 04: **[Introduce a High Availability with Central Network Virtual Appliances](./Solution-04.md)**
+	 - High Availability concepts
+
+## Coach Prerequisites
+
+This hack has pre-reqs that a coach is responsible for understanding and/or setting up BEFORE hosting an event. Please review the [What The Hack Hosting Guide](https://aka.ms/wthhost) for information on how to host a hack event.
+
+The guide covers the common preparation steps a coach needs to do before any What The Hack event, including how to properly configure Microsoft Teams.
+
+### Student Resources
+
+Before the hack, it is the Coach's responsibility to download and package up the contents of the `/Student/Resources` folder of this hack into a "Resources.zip" file. The coach should then provide a copy of the `Resources.zip` file to all students at the start of the hack. 
+
+In Challenge 00, the students will run a script from this Resources.zip file that deploys a baseline Hub & Spoke network topology into their Azure subscriptions with Linux VMs.  
+
+The coach should provide the students with the Username and password used for the Ubuntu VMs:
+- username: azureuser
+- psswd: Msft123Msft123
+
+**NOTE:** The script deploys Active/Active VPNs with BGP and the correspondent Vnet Peering attributes for transitivity. However, other aspects such as configuring Local Network Gateways, setting up required Route Tables (UDRs) will need to be done manually. Simulated on-premises and Central NVA templates are provided separately throughout the challenge
+
+Always refer students to the [What The Hack website](https://aka.ms/wth) for the student guide: [https://aka.ms/wth](https://aka.ms/wth)
+
+**NOTE:** Students should **not** be given a link to the What The Hack repo before or during a hack. The student guide does **NOT** have any links to the Coach's guide or the What The Hack repo on GitHub.
+
+### Additional Coach Prerequisites
+
+The coach should offer the student the aforementioned script file at the start of the event so credits using the VPN Gateway are not consumed before the Hack. If the student opts to do the initial setup manually, bring awareness that this is going to consume time and is prone to errors. 
+
+The script takes about 30 minutes to deploy. In the meantime, the coach can proceed with a lecture or explanation of the challenges. 
+
+## Azure Requirements
+
+This hack requires students to have access to an Azure subscription where they can create and consume Azure resources. These Azure requirements should be shared with a stakeholder in the organization that will be providing the Azure subscription(s) that will be used by the students.
+
+_Please list Azure subscription requirements._
```

Comment:
This section of the template for the Coach's guide needs to be filled out!  You need to provide information here that can be shared with a stakeholder in the student's organization so that they know what is needed for the Azure subscriptions students will need to have access to.

List out resources that will be consumed by each student (so a stakeholder can approximate costs).
List out which Azure permissions are needed by each student (owner or contributor?).  Many organizations are not willing to provide students with "owner" access on a subscription by default.   If "owner" access IS needed, than this is something the stakeholder will need to plan for and work within their organization to get a proper sandbox subscription provided.